feature/empty splash screen

keithchad opened the pull request:
I added splash.view package then added a Kotlin class for the Splash screen. I then added it to Screens then I changed the first screen to Splash screen and added a two second delay then it navigates to Home screen.
Closes #19

chepsi reviewed (changes requested):
Can we use [splash api](https://developer.android.com/guide/topics/ui/splash-screen) instead.

keithchad commented:
> Can we use [splash api](https://developer.android.com/guide/topics/ui/splash-screen) instead.

Yea sure. Working on it.

keithchad commented:
> Can we use [splash api](https://developer.android.com/guide/topics/ui/splash-screen) instead.

I added the [splash api](https://developer.android.com/guide/topics/ui/splash-screen) dependency to presentation Gradle file then I then I created a theme for the Splash screen then I added a drawable icon for the animated icon then I called installSplashScreen in MainActivity.kt

chepsi reviewed (approved):
LGTM

jumaallan commented on presentation/build.gradle.kts:
```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation(libs.bundles.compose)
     implementation(libs.lifecycle.runtimeKtx)
     implementation(libs.android.hilt)
+    implementation("androidx.core:core-splashscreen:1.0.0-rc01")
```
hey @keithchad Can you define this dependency like the others

Basically:
1. Look for `libs.versions.toml`, and define this dependency there (top of the file [versions] defines the version, then add it under the [libraries] section) - see how other dependencies are setup
2. Then use it on the presentation build file like ` implementation(libs.androix.splashscreen)`

keithchad replied:
Okay. Working on it.

keithchad replied:
I have updated both Gradle files with the right implementation.

jumaallan replied:
Thank you

jumaallan reviewed (changes requested):
Left some small review
